digiroad2-geo/src/main/scala/fi/liikennevirasto/digiroad2/GeometryUtils.scala

```diff
@@ -381,6 +391,68 @@ object GeometryUtils {
     ((projection.oldEnd - projection.oldStart)*(projection.newEnd - projection.newStart)) < 0
   }
 
+  def pointsToLineString(points: Seq[Point]): LineString = {
+    val coordinates = points.map(p => new Coordinate(p.x, p.y)).toArray
+    val lineString = geometryFactory.createLineString(coordinates)
+    lineString
+  }
+
+  /**
+    * Measures the degree of similarity between two Geometries using the Hausdorff distance metric.
+    * The measure is normalized to lie in the range [0, 1]. Higher measures indicate a great degree of similarity.
+    * @param geom1
+    * @param geom2
+    * @return Measure of similarity from 0 to 1
+    */
+  def getHausdorffSimilarityMeasure(geom1: LineString, geom2: LineString): Double = {
+    hausdorffSimilarityMeasure.measure(geom1, geom2)
+  }
+
+  /**
+    * Measures the degree of similarity between two Geometries
+    * using the area of intersection between the geometries.
+    * The measure is normalized to lie in the range [0, 1].
+    * Higher measures indicate a great degree of similarity.
+    *
+    * When used with Road Links, use buffer
+    * @param geom1 geometry 1 to compare
+    * @param geom2 geometry 2 to compare
+    * @return Normalized measure in range [0, 1] measuring the similarity of given geometries.
+    *         0 = No intersection, 1 = Full intersection
+    */
+  def getAreaSimilarityMeasure(geom1: JtsGeometry, geom2: JtsGeometry): Double = {
+    areaSimilarityMeasure.measure(geom1, geom2)
+  }
+
+  /**
+    * Calculates the centroid (average of all the points) of given points
+    * @param points Points to calculate centroid from
+    * @return calculated centroid as a Point object
```

Comment:
in layman term center point

Reply:
Käsittääkseni painopiste (engl. centroid) on tässä oikea termi